dm/tests/openapi/client/openapi_task_check

```diff
@@ -413,6 +413,73 @@ def get_task_status_success(task_name, total):
     print("get_task_status_success resp=", data)
     assert data["total"] == int(total)
 
+def check_sync_task_status_success(
+    task_name,
+    min_dump_io_total_bytes=0,
+    max_dump_io_total_bytes=sys.maxsize,
+    min_io_total_bytes=0,
+    max_io_total_bytes=sys.maxsize
```

Comment:
> The value sys.maxsize, on the other hand, reports the platform's pointer size, and that limits the size of Python's data structures such as strings and lists.

maybe use a more restricted range, such as a few KiB or MiB

Reply:
what about the minimum? i guess we will at least send some rows for DM to sync. we want to check that the `bytes` are in a reasonable range to meter correct

Reply:
For sure, for now the average of minimum is 3KiB. I think we can set to 2KiB for more stable test. And maximum set to 50KiB.